android/src/main/java/com/tadasr/IOTWifi/IOTWifiModule.java

```diff
@@ -97,9 +104,66 @@ private void connectToWifi(String ssid, String passphrase, Boolean isWEP, Callba
         }
     }
 
+    private void bindToNetwork(final String ssid) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
I think fast fail should be preferred over bing enclosing If conditions. Reverse the condition
```
if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
 return;
}
```